main.py

```diff
@@ -126,53 +112,61 @@ def main(args):
         model = torch.nn.parallel.DistributedDataParallel(model, device_ids=[args.gpu])
         model_without_ddp = model.module
     n_parameters = sum(p.numel() for p in model.parameters() if p.requires_grad)
+    params = [p for n, p in model_without_ddp.named_parameters() if "backbone" not in n and p.requires_grad]
```

Comment:
@ezekielbarnett this might be wrong -- in the original they set a different learning rate for `"backbone" not in n` and `"backbone" in n`.  Your version sets the same LR for both